config/tls/networking-certmanager.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 The Knative Authors
```

Comment:
I'd still expect these files directly in `config/`, but labeled so that we can filter them.

Reply:
I'd also like to see a directory like `third_party/istio...` and an update to DEVELOPMENT.md discussing these.

Reply:
I'm fine with that, but can we do that first?  My concern is that `ko apply -f config/` will start failing for folks when this goes in.  We should also update the e2e tests to install cert-manager.

Otherwise this LGTM :)

Reply:
At this point, I updated the doc and test script to only install `cert-manager-crds.yaml` as the auto TLS feature has not been landed.
Will update all of the docs and test scripts to install the full `cert-manager.yaml` when this feature is finished.